src/blocks/ExtendedFeatures/ExtendedFeatures.tsx

```diff
@@ -30,7 +30,7 @@ export const ExtendedFeaturesBlock = ({
 
     return (
         <AnimateBlock className={b()} animate={animated}>
-            <BlockHeader title={title} description={description} className={b('header')} />
+            <Title title={title} description={description} className={b('header')} />
```

Comment:
is it strange that component `Title` has property `title`?) Whole component is title
Or in this case `description` should became `subtitle` may be

Reply:
I renamed `description` to `subtitle`